### PR TITLE
Include more variations of closed statuses

### PR DESCRIPTION
**Pull request by zcorpan**
e.g. VERIFIED FIXED

### CODE REVIEW

**Comment by zcorpan** [comment]
Company name: Opera Software ASA

File: js/app.js [hunk 1]
```diff
@@ -36,14 +36,14 @@ const domains_to_browser = {
...
 
-const CLOSED_STATUSES = [
-  'FIXED',
-  'WONTFIX',
-  'CLOSED',
-  'RESOLVED',
-  'RESOLVED FIXED',
-  'DUPLICATE'
-];
+const STATUS = ['RESOLVED', 'VERIFIED', 'CLOSED'];
```
Comment (ebidel) [inline js/app.js]: This seems overly complicated for a few times. Let's just keep the array and add the extra item :)

Comment (zcorpan) [inline js/app.js]: Better? :-)

**Comment by ebidel** [comment]
👍 thank you!

**Pull request merged**